src/libraries/restore/runtime/runtime.depproj

```diff
@@ -18,8 +18,8 @@
   </ItemGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.NETCore.DotNetHost" Version="$(MicrosoftNETCoreDotNetHostVersion)" />
-    <PackageReference Include="Microsoft.NETCore.DotNetHostPolicy" Version="$(MicrosoftNETCoreDotNetHostPolicyVersion)" />
+    <PackageReference Condition="'$(BinPlaceTestRuntimePack)' != 'true'" Include="Microsoft.NETCore.DotNetHost" Version="$(MicrosoftNETCoreDotNetHostVersion)" />
```

Comment:
Could you move the condition to the ItemGroup instead?

```xml
<ItemGroup Condition="'$(BinPlaceTestRuntimePack)' != 'true'">
   ...
</ItemGroup>
```

Reply:
Done.